src/author.rs

```diff
@@ -28,7 +28,7 @@ impl Author {
     /// The getter method for the field `name` of the corresponding struct.
     ///
     /// Since the struct `Author` defines its field `name` to be private, this
-    /// getter is utilised in order to operate on this field.  It returns a
+    /// getter is used in order to operate on this field.  It returns a
```

Comment:
This should have been squashed with the commit, that introduced the first change from utilised to used. (See my comment there)